test/Common.h

```diff
@@ -83,4 +83,6 @@ struct CommonOptions: boost::noncopyable
 	static std::unique_ptr<CommonOptions const> m_singleton;
 };
 
+bool isValidSemanticTestPath(boost::filesystem::path const& _testPath);
```

Comment:
This function needs some documentation:

```suggestion
/// @return true if it is ok to treat the file located under the specified path as a semantic test.
/// I.e. if the test is located in the semantic test directory and is not excluded due to being a part of extenal sources.
/// Note: @p _testPath can be relative but must include at least the `/test/libsolidity/semanticTests/` part.
bool isValidSemanticTestPath(boost::filesystem::path const& _testPath);
```